src/ngraph/env_util.hpp

```diff
@@ -18,28 +18,117 @@
 
 #include <string>
 
+#include "ngraph/ngraph_visibility.hpp"
```

Comment:
I added env_util.hpp to ngraph so that the NNP could use this version, but now you have added code that makes it so that I can no longer use this file. Please move the env cache to its own files so that env_util.?pp files are generic and contains no ngraph nor backend specific variables. It should just be the three original functions to get int, bool, and string env vars.

Reply:
ok, will move cache and registry it to it's own file.